src/test/java/uk/gov/companieshouse/mapper/AppealMapperTest.java

```diff
@@ -38,40 +57,27 @@ void shouldReturnNullWhenValueIsNull() {
 
         @Test
         void shouldMapValueWhenValueIsNotNull() {
-            Reason reason = new Reason();
-            reason.setOther(new OtherReason(
-                TestData.Appeal.Reason.OtherReason.title,
-                TestData.Appeal.Reason.OtherReason.description,
-                Lists.newArrayList(new Attachment(
-                    TestData.Appeal.Reason.Attachment.id,
-                    TestData.Appeal.Reason.Attachment.name,
-                    TestData.Appeal.Reason.Attachment.contentType,
-                    TestData.Appeal.Reason.Attachment.size,
-                    null
-                ))
-            ));
+            when(createdByMapper.map(any(CreatedBy.class))).thenReturn(null);
+            when(penaltyIdentifierMapper.map(any(PenaltyIdentifier.class))).thenReturn(penaltyIdentifierEntity);
+            when(reasonMapper.map(any(Reason.class))).thenReturn(reasonEntity);
+
             AppealEntity mapped = mapper.map(new Appeal(
                 null,
                 null,
-                null,
-                new PenaltyIdentifier(
-                    TestData.Appeal.PenaltyIdentifier.companyNumber,
-                    TestData.Appeal.PenaltyIdentifier.penaltyReference
-                ),
-                reason
+                mockCreatedBy,
+                mockPenaltyIdentifier,
+                mockReason
             ));
+
             assertNull(mapped.getId());
             assertNull(mapped.getCreatedAt());
             assertNull(mapped.getCreatedBy());
```

Comment:
We could probably improve the tests by not using values for these fields.
Or adding a new test where they aren't null.
May be out of scope for a refactor.